shared/js/background/chrome-events.es6.js

```diff
@@ -167,12 +167,12 @@ const redirect = require('./redirect.es6')
 const tabManager = require('./tab-manager.es6')
 const pixel = require('./pixel.es6')
 const https = require('./https.es6')
-const cookieConfig = require('./../background/storage/cookies.es6')
+const configStorage = require('./../background/storage/config.es6')
 
 const requestListenerTypes = utils.getUpdatedRequestListenerTypes()
 
 function blockTrackingCookies () {
-    return true
+    return configStorage.config.features?.cookieProtections.state === 'enabled' || true
```

Comment:
I think this will always resolve true.

Reply:
Yeah I was afraid of this. I'll fix this one.